families/track_and_trade/sawtooth_track_and_trade/handler.py

```diff
@@ -0,0 +1,633 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+import logging
+import time
+
+from sawtooth_sdk.processor.state import StateEntry
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from sawtooth_sdk.processor.exceptions import InternalError
+from sawtooth_sdk.protobuf.transaction_pb2 import TransactionHeader
+
+from sawtooth_track_and_trade.protobuf.agent_pb2 import Agent, AgentContainer
+from sawtooth_track_and_trade.protobuf.property_pb2 import \
+    Property, PropertyContainer, PropertyPage, PropertyPageContainer, \
+    PropertySchema, PropertyValue
+from sawtooth_track_and_trade.protobuf.proposal_pb2 import \
+    Proposal, ProposalContainer
+from sawtooth_track_and_trade.protobuf.record_pb2 import \
+    Record, RecordContainer, RecordType, RecordTypeContainer
+from sawtooth_track_and_trade.protobuf.payload_pb2 import \
+    TTPayload, CreateAgentAction, CreateRecordAction, FinalizeRecordAction, \
+    CreateRecordTypeAction, UpdatePropertiesAction, CreateProposalAction, \
+    AnswerProposalAction, RevokeReporterAction
+
+import sawtooth_track_and_trade.addressing as Addressing
+
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+
+
+PROPERTY_PAGE_MAX_LENGTH = 256
+
+
+class TrackAndTradeTransactionHandler:
+    @property
+    def family_name(self):
+        return Addressing.FAMILY_NAME
+
+    @property
+    def family_versions(self):
+        return ['1.0']
```

Comment:
I love global constants as much as the next person (maybe even more), but since `family_versions` and `encodings` don't have any effect on anything else, I'd prefer to leave them there. (That's also how all the other TPs do it afaik.)